translations/ui-users/en.json

```diff
@@ -852,6 +854,8 @@
   "permission.settings.departments.all": "Settings (Users): Create, edit, view, and delete departments",
   "permission.overridePatronBlock":"User: Can override patron blocks",
   "permission.overrideItemBlock":"User: Can override item blocks",
```

Comment:
Since you are here, could you please change `User:` --> `Users:` on lines 855, 856? And in `package.json` file as well on lines 760, 768

Reply:
We should discuss current changes with product owner and only after approval we should create task for adding changes to translation file.